docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -52,29 +52,36 @@ image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
 [float]
 [[view-alert-details]]
 === View alert details
-To further inspect an alert, click the *View details* button from the Alerts table. The Alert details flyout appears and provides several options to view alert data.
+To further inspect an alert, click the *View details* button from the Alerts table. The alert details flyout appears with several options to view alert data.
```

Comment:
Tweak to help re-establish the type of alert we're talking about, since this is a long page:
```suggestion
To further inspect a detection alert, click the *View details* button from the Alerts table. The alert details flyout appears with several options to view alert data.
```